src/test-data.js

```diff
@@ -0,0 +1,30 @@
+const prototypeTestData = [{
```

Comment:
Followed the advice from Hannah to create a separate data list to use solely for testing.